web-bundle/src/main/resources/com/graphhopper/maps/js/config/options.js

```diff
@@ -11,7 +11,7 @@
 exports.options = {
     with_tiles: true,
     environment: "development",
-    // for local development use this (assuming your GH server is running on the standard port)
+    // if you server GH maps from another server than the GH server use this (assuming your GH server is running on the standard port)
```

Comment:
minor typo: server -> serve ?

Reply:
Ok I wrote: 'Use this if you *serve* the web UI from ...'